src/SUMMARY.md

```diff
@@ -2,41 +2,17 @@
 
 - [Sobre o Livro](./about.md)
 - [Licença](./license.md)
-- [Começando](./ch01-comecando/index.md)
-  - [Instalando uma implementação](./ch01-comecando/ch01-01_instalando.md)
-  - [Iniciando um REPL](./ch01-comecando/ch01-02_iniciando-repl.md)
-  - [*Libraries* (bibliotecas)](./ch01-comecando/ch01-03_bibliotecas.md)
-	- [Terminologia](./ch01-comecando/ch01-04_terminologia.md)
-	- [Instalando o Quicklisp](./ch01-comecando/ch01-05_quicklisp.md)
-	- [Instalando *libraries*](./ch01-comecando/ch01-06_bibliotecas.md)
-	- [Gerenciamento avançado de dependências](./ch01-comecando/ch01-07_dependencias.md)
-  - [Trabalhando com *Projects*](./ch01-comecando/ch01-08_projetos.md)
-    - [Criando um novo *project*](./ch01-comecando/ch01-09_criando-project.md)
-	- [Como carregar um *project* existente](./ch01-comecando/ch01-10_carregando-project.md)
-	- [Mais configurações](./ch01-comecando/ch01-11_mais-config.md)
-	- [Leia Mais](./ch01-comecando/ch01-12_leia-mais.md)
-	- [Créditos](./ch01-comecando/ch01-13_creditos.md)
-- [Suporte de Editores de Texto](./null.md)
-- [Estruturas de Dados](./null.md)
-- [Strings](./null.md)
-- [Datas e Tempo](./null.md)
-- [Pattern Matching](./null.md)
-- [Expressões Regulares](./null.md)
-- [Funções](./null.md)
-- [Loop](./null.md)
-- [Entrada/Saída](./null.md)
-- [Arquivos e Diretórios](./null.md)
-- [Erros e manipulação de condições](./null.md)
-- [Pacotes](./null.md)
-- [Macros e Backquote](./null.md)
-- [CLOS (O Common Lisp Object System)](./null.md)
-- [Fazendo interface com seu SO](./null.md)
-- [Interfaces de Foreign Function (FFI)](./null.md)
-- [Threads](./null.md)
-- [Definindo Sistemas](./null.md)
-- [Usando a API Win32](./null.md)
-- [Scripting, Construindo executáveis](./null.md)
-- [Testes e Integração Contínua](./null.md)
-- [Web Scraping](./null.md)
-- [CL21 Experimental](./null.md)
-- [Miscelânea](./null.md)
+- [Começando](./ch01/index.md)
+  - [Instalando uma implementação](./ch01/ch01-01_instalando.md)
+  - [Iniciando um REPL](./ch01/ch01-02_iniciando-repl.md)
+  - [*Libraries* (bibliotecas)](./ch01/ch01-03_bibliotecas.md)
+	- [Terminologia](./ch01/ch01-04_terminologia.md)
+	- [Instalando o Quicklisp](./ch01/ch01-05_quicklisp.md)
```

Comment:
Parece que ficou bem mais claro assim, gostei também do uso do itálico pra termos em outra língua